content/1.2-rc/docs/serverless-workflows/troubleshooting.md

```diff
@@ -116,6 +117,12 @@ In this configuration:
 
 By following these steps, the workflow will have the necessary credentials to access PostgreSQL and will correctly reference the service in a different namespace.
 
+## HTTP Errors
+Many workflow operations are REST requests to HTTP servers. If an HTTP error occurs then the workflow will fail and the HTTP code and message will be displayed. Please use [HTTP codes documentation](https://developer.mozilla.org/docs/Web/HTTP/Status) for understanding the meaning of such errors.  
```

Comment:
> the HTTP code and message will be displayed

it's only displayed in the UI on the node that failed, right?

Reply:
I would show an example of where/how the error is displayed

Reply:
I don't have an example. besides, what is there to show? user will see the HTTP status code

Reply:
I would be good to specify where the error will be displayed: in the node that fails as a tooltip, in the status as a tooltip, in a dedicated place... there are so many options for the user to look
I think it's displayed in the node that failed as a tooltip when the node state is hovered

Reply:
fixed